index.html

```diff
@@ -11,8 +11,10 @@
     />
     <link rel="icon" type="image/x-icon" href="./assets/ma_logo.png" />
     <link href="./stylesheets/pages/index.css" rel="stylesheet" />
+    <link href="./stylesheets/pages/login-signup.css" rel="stylesheet" />
```

Comment:
just a small suggestion that you can move the stylesheet in one main CSS file the you only need to import only one CSS file in the header file

Reply:
changes done, thanks